app/src/main-process/menu/build-default-menu.ts

```diff
@@ -297,11 +297,16 @@ export function buildDefaultMenu(
         click: emit('merge-branch'),
       },
       separator,
+      {
+        label: __DARWIN__ ? 'Compare on Desktop' : '&Compare on Desktop',
```

Comment:
Also I'm still a huge fan of `Compare to Branch` here to avoid excessive mentions of `Desktop` in the app.